.github/CODEOWNERS

```diff
@@ -10,6 +10,7 @@ yarn.lock                                           @backstage/community-plugins
 
 /workspaces/3scale                                  @backstage/community-plugins-maintainers  @04kash @AndrienkoAleksandr
 /workspaces/acr                                     @backstage/community-plugins-maintainers  @christoph-jerolimov @ciiay @invincibleJai
+/workspaces/acs                                     @backstage/community-plugins-maintainers  @sachaudh @alwayshooin @dvail @maknop
```

Comment:
Sorry, I wasn't thinking about this before but only you will get credit for this work and get Org Member access @maknop, you need to actually have some sort of contribution for that which you have done but the others have not. No actions needed on this file, just raising awareness.

Reply:
No, they need to make actual contributions to the repo to get Org Member status. Let's cover this in our next Community Plugin SIG. 👍 

This won't block merging once all the comments are settled and CI is green.